### PR TITLE
Added exercise space-age (see #3)

### PR DESCRIPTION
**Pull request by pietvandongen**

### CODE REVIEW

**Comment by kytrinyx** [comment]
Awesome thank you!

**Comment by pietvandongen** [comment]
@kytrinyx Awesome, thank you too! I'll do more when I have some time in between stuff.

**Comment by kytrinyx** [comment]
That would be great. Right now Java, Objective-C and PHP are the languages with the fewest exercises.

File: space-age/example.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        EARTH, MERCURY, VENUS, MARS, JUPITER, SATURN, URANUS, NEPTUNE
+    }
+
+    private double seconds;
+
+    private static final double EARTH_ORBITAL_PERIOD_IN_SECONDS = 31557600.0;
+    private static final int PRECISION = 2;
+    private static final Map<Planet, Double> relativeOrbitalPeriods = new HashMap<>();
+
+    static {
```
Comment (sit) [inline space-age/example.java]: Could fold this into the `Planet` enum itself. (Oddly enough, it's in the official tutorial :-)

https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

**Comment by sit** [comment]
@pietvandongen hope you don't mind all the comments. Thanks for picking up the Java and even Java 8!

**Comment by pietvandongen** [comment]
@sit I don't mind, thanks! I'm not sure if I can / will respond to all of them, but it's good to have these comments here anyway :)

**Pull request merged**